<ui version="4.0" >
 <class>Salat</class>
 <widget class="QMainWindow" name="Salat" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>300</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" />
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QWidget" native="1" name="widget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string notr="true" >background-image: url(:/images/qsalat.png);</string>
    </property>
    <widget class="QLabel" name="label_fajr" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>100</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Arial Black</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet" >
      <string notr="true" >color:white;
opacity:1.0;
background-image: url(:/images/slider3.png);</string>
     </property>
     <property name="text" >
      <string>06:00</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_asr" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>190</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Arial Black</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet" >
      <string notr="true" >color:white;
opacity:1.0;
background-image: url(:/images/slider3.png);</string>
     </property>
     <property name="text" >
      <string>00:00</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_isha" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>280</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Arial Black</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet" >
      <string notr="true" >color:white;
opacity:1.0;
background-image: url(:/images/slider3.png);</string>
     </property>
     <property name="text" >
      <string>00:00</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_duhr" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>145</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Arial Black</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet" >
      <string notr="true" >color:white;
opacity:1.0;
background-image: url(:/images/slider3.png);</string>
     </property>
     <property name="text" >
      <string>06:00</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_maghreb" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>235</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Arial Black</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet" >
      <string notr="true" >color:white;
opacity:1.0;
background-image: url(:/images/slider3.png);</string>
     </property>
     <property name="text" >
      <string>06:00</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_location" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>340</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet" >
      <string notr="true" >background-image: url(:/images/slider3.png);
color:white;</string>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_hijri" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>20</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Arab</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet" >
      <string notr="true" >background-image: url(:/images/slider3.png);
color:white;
</string>
     </property>
     <property name="text" >
      <string>00 Jumada al-Thani 0000</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionHide" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionHijri_date" />
    <addaction name="actionQibla_direction" />
    <addaction name="actionLocation" />
    <addaction name="actionAudio" />
    <addaction name="actionCalculation_methods" />
    <addaction name="actionPrayers_time_option" />
   </widget>
   <widget class="QMenu" name="menuExtra" >
    <property name="title" >
     <string>Extra</string>
    </property>
    <addaction name="actionWorld_time_2" />
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>About</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionAbout_Qsalat" />
    <addaction name="actionAbout_QT" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuTools" />
   <addaction name="menuExtra" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionHide" >
   <property name="icon" >
    <iconset resource="../salat.qrc" >
     <normaloff>:/images/mecque.png</normaloff>:/images/mecque.png</iconset>
   </property>
   <property name="text" >
    <string>Hide</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHijri_date" >
   <property name="text" >
    <string>Hijri date</string>
   </property>
  </action>
  <action name="actionQibla_direction" >
   <property name="text" >
    <string>Qibla direction</string>
   </property>
  </action>
  <action name="actionLocation" >
   <property name="text" >
    <string>Location</string>
   </property>
  </action>
  <action name="actionAudio" >
   <property name="text" >
    <string>Audio</string>
   </property>
  </action>
  <action name="actionCustom_option" >
   <property name="text" >
    <string>Custom option</string>
   </property>
  </action>
  <action name="actionWorld_time" >
   <property name="text" >
    <string>World time</string>
   </property>
  </action>
  <action name="actionAbout_Qsalat" >
   <property name="text" >
    <string>About Qsalat</string>
   </property>
  </action>
  <action name="actionAbout_QT" >
   <property name="text" >
    <string>About QT</string>
   </property>
  </action>
  <action name="actionCalculation_methods" >
   <property name="text" >
    <string>Calculation methods</string>
   </property>
  </action>
  <action name="actionWorld_time_2" >
   <property name="text" >
    <string>World time</string>
   </property>
  </action>
  <action name="actionPrayers_time_option" >
   <property name="text" >
    <string>Prayers time option</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../salat.qrc" />
 </resources>
 <connections/>
</ui>
